### ISSUE TITLE
setup.py install doesn't work on PyPy

### ISSUE THREAD

**Issue by yakkis** [abusesa/abusehelper#61]
When installing AbuseHelper using PyPy (pypy setup.py install) the script doesn't find the needed packages. The problem lies in the function `is_package()` which is called when collecting packages in function `collect_package()`.

The problematic function below:

```
def is_package(path):
    try:
        imp.find_module(".", [path])
    except ImportError:
        return False
    return True
```

This will always raise ImportError on PyPy with a message `ImportError: No module named .` The result is that the actual code is not copied to site-packages folder, only the `abusehelper-*.egg-info` file appears. On Python the same function seems to work, returning something like `(None, 'abusehelper/.', ('', '', 5))`.

**Comment by yakkis** [comment]
The newer setup.py in branch setuptoolsify seems to install fine on PyPy.

**Comment by oherrala** [comment]
Apparently related to pull request #37

**Comment by oherrala** [comment]
Fixed in #37. Closing

**Comment by kevien** [comment]
how to install ？
